recipes/iceoryx/all/conanfile.py

```diff
@@ -0,0 +1,244 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import textwrap
+
+class IceoryxConan(ConanFile):
+
+    name = "iceoryx"
+    version = "1.0.0"
```

Comment:
```suggestion
```

Keep the recipe generic. The user and CI should pass the version according the conandata.yml